homeassistant/components/meater/sensor.py

```diff
@@ -10,10 +21,112 @@
     CoordinatorEntity,
     DataUpdateCoordinator,
 )
+from homeassistant.util import dt as dt_util
 
 from .const import DOMAIN
 
 
+@dataclass
+class MeaterSensorEntityDescription(SensorEntityDescription):
+    """Describes meater sensor entity."""
+
+    available: Callable[
+        [MeaterProbe | None], bool | type[NotImplementedError]
+    ] = lambda x: NotImplementedError
```

Comment:
Isn't it more clear to implement a mixin dataclass where these attributes are required, than setting a default which raises?

Reply:
Improved in #71283